website/docs/deployment.mdx

```diff
@@ -150,6 +150,12 @@ This workflow assumes your documentation resided in `documentation` branch of yo
 1. Copy your private key with `xclip -sel clip < ~/.ssh/id_rsa` and paste a GitHub secret with name `GH_PAGES_DEPLOY`. Copy file content if the command line doesn't work for you. Save your secret.
 1. Create you [documentation workflow file](https://help.github.com/en/actions/configuring-and-managing-workflows/configuring-a-workflow#creating-a-workflow-file) in `.github/workflows/`. In this example it's `documentation.yml`.
 
+:::warning
+
+Please make sure that you replace `actions@github.com` with your github email and `gh-actions` with your name
```

Comment:
```suggestion
Please make sure that you replace `actions@github.com` with your GitHub email and `gh-actions` with your name.
```

Reply:
Done: [0aae91](https://github.com/facebook/docusaurus/pull/4876/commits/0aae91dfb0aa93770fa843f7356fd0669efe7634)

Reply:
I think you missed this one, the current file doesn't seem to have this change. 😉